team_test.go

```diff
@@ -139,6 +136,59 @@ func TestTeamsRead(t *testing.T) {
 	})
 }
 
+func TestTeamsUpdate(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	teamTest, teamTestCleanup := createTeam(t, client, orgTest)
```

Comment:
`teamTest` is called `tmTest` everywhere else, so I guess we should do that here as well to keep things consistent.